src/main.cpp

```diff
@@ -1078,14 +1155,21 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
     if (!CheckTransaction(tx, state, verifier))
         return error("AcceptToMemoryPool: CheckTransaction failed");
 
+    // FIXME: DoS level set to 10 to be more forgiving... or use 100?
```

Comment:
I think we want to be forgiving here (as mempool transactions could race with the activation block), but not *too* forgiving (for DoS prevention).

Reply:
Delete "FIXME:" and "... or use 100?".